.markdownlint.jsonc

```diff
@@ -30,8 +30,12 @@
       "script"
     ]
   },
+  // This warns if emails are used without '<' and '>'. That's ok.
+  "MD034": false,
   // This warns if you have spaces in code blocks. Sometimes, that's fine.
   "MD038": false,
+  // This warns if there's no H1 at the first line. Disabled because of README.md.
+  "MD041": false,
```

Comment:
@paulo-ferraz-oliveira does this get fixed if we wrap the first `<img>` in the README inside a `<h1>`?

Reply:
@whatyouhide Tried that, but then GitHub would insert the heading border twice. There's no way to hide one the h1 depending on the dark/light theme.

Reply:
omg, I could try to wrap both imgs in the same h1 🤦🏻